tests/django/test_scalars.py

```diff
@@ -5,10 +5,13 @@
 from ariadne.contrib.django.scalars import (
     date_scalar,
     datetime_scalar,
+    time_scalar,
```

Comment:
Please sort imports alphabetically - `time_scalar` should go under `serialize_time`.

Reply:
Done! Maybe it would be good to make travis run `isort`?